### ISSUE TITLE
Is there any way to change resolution or bitrate?

### ISSUE THREAD

**Issue by thegrandmasterflash** [dunnousername/yanderifier#84]
Is there any way to change resolution or bitrate?  The output videos are square and extremely low qualty.  Is there any way to adjust that in some hidden settings file I don't know about?

**Comment by Animan8000** [comment]
Yes. So basically [this line](https://github.com/dunnousername/yanderifier/blob/7f7608ac3150378f0820e8a251f0166971f7a98a/yanderify.py#L240) causes the video to be in a set (and rather low) constant bitrate instead of lossless. Which makes it even worse, is the fact that all videos are always 256x256 (1:1 format) initially and if your image has a higher resolution, then it will be upscaled, giving it a blurry effect (not always necessarily bad, especially hair doesn't look too pixelated in some cases (making it look more *real*)) but also requiring a higher bitrate, where this set constant bitrate kills the quality even further. I have a [fork](https://github.com/Animan8000/Yanderify-bit-rate-fix) which is slightly outdated but generates outputs in lossless quality, in terms of bitrate.

Also worth noting that the v5 alpha generates lossless outputs as well as no upscaling IIRC, in case you don't want that in your deepfakes.

Other than that, for real higher resolution I'm pretty sure it requires a different dataset and some internal code changes (pretty sure without code changes it would just crash or cause other accuracy issues).

If you intend uploading a deepfake on YouTube, then I'd recommend at least 1080x1080 resolution because YouTube's video processing on low resolution videos is absolutely **terrible** and kills the quality even further.
